pkg/operator/ceph/disruption/clusterdisruption/osd.go

```diff
@@ -97,7 +97,7 @@ func (r *ReconcileClusterDisruption) createOverallPDBforOSD(namespace string) er
 	err = r.client.Get(context.TODO(), pdbRequest, &policyv1beta1.PodDisruptionBudget{})
 	if err != nil {
 		if apierrors.IsNotFound(err) {
-			logger.Info("all osds are up. pg health is active+clean")
```

Comment:
removing `all osds are up` log message since we are ignoring disk failures as node drain events.

Reply:
is the "pg health is active+clean" still true as well? If a single osd is down from a disk failure, the PGs wouldn't be clean, right?

Reply:
yes. reverted this change.